templates/logsearch-for-cf.example-with-uaa-auth.yml

```diff
@@ -49,6 +49,20 @@ jobs:
         system_domain: sys.cf.example
         admin_client_secret: admin_client_secret
 
+- name: upload-kibana-dashboards
+  lifecycle: errand
+  release: logsearch-for-cloudfoundry
+  instances: 1
+  templates:
+  - {name: upload-kibana-dashboards, release: logsearch-for-cloudfoundry}
+  networks:
+  - name: default
+  resource_pool: errand
+  properties:
+    elasticsearch:
+      host: elastic_host
```

Comment:
Hi @jmcarp 
It would be great if we could avoid manual actions in stub filling.
I would suggest to replace the value with `(( grab jobs.elasticsearch_master.networks.default.static_ips.[0] ))`.
Thanks!